egs/aishell/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# This script is modified based on swbd/s5c/local/nnet3/run_ivector_common.sh
+
+# this script contains some common (shared) parts of the run_nnet*.sh scripts.
+
+. cmd.sh
+
+
+stage=0
+num_threads_ubm=32
+
+set -e
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+gmm_dir=exp/tri5a
+align_script=steps/align_fmllr.sh
+
+if [ $stage -le 1 ]; then
+  # Create high-resolution MFCC features (with 40 cepstra instead of 13).
+  # this shows how you can split across multiple file-systems.  we'll split the
+  # MFCC dir across multiple locations.  You might want to be careful here, if you
+  # have multiple copies of Kaldi checked out and run the same recipe, not to let
+  # them overwrite each other.
+  mfccdir=mfcc_hires
+  if [[ $(hostname -f) == *.clsp.jhu.edu ]] && [ ! -d $mfccdir/storage ]; then
+    utils/create_split_dir.pl /export/b0{5,6,7,8}/$USER/kaldi-data/egs/aishell-$(date +'%m_%d_%H_%M')/s5/$mfccdir/storage $mfccdir/storage
+  fi
+
+  for datadir in train dev test; do
+    utils/copy_data_dir.sh data/$datadir data/${datadir}_hires
+    if [ "$datadir" == "train" ]; then
+      utils/data/perturb_data_dir_volume.sh data/${datadir}_hires
+    fi
+
+    steps/make_mfcc_pitch.sh --nj 70 --mfcc-config conf/mfcc_hires.conf \
+      --cmd "$train_cmd" data/${datadir}_hires exp/make_hires/$datadir $mfccdir || exit 1;
+    steps/compute_cmvn_stats.sh data/${datadir}_hires exp/make_hires/$datadir $mfccdir || exit 1;
+
+    # make MFCC data dir without pitch to extract iVector
+    utils/data/limit_feature_dim.sh 0:39 data/${datadir}_hires \
+        data/${datadir}_hires_nopitch || exit 1;
+    steps/compute_cmvn_stats.sh data/${datadir}_hires_nopitch \
+        exp/make_hires/$datadir $mfccdir || exit 1;
+  done
+fi
+
+if [ $stage -le 2 ]; then
+  # Train a system just for its LDA+MLLT transform.  We use --num-iters 13
+  # because after we get the transform (12th iter is the last), any further
+  # training is pointless.
+  steps/train_lda_mllt.sh --cmd "$train_cmd" --num-iters 13 \
```

Comment:
can you try replacing this with the pca transform training -- the parameters are the same (if you decide not to go for the minils ivector-common script)